src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -1032,7 +957,7 @@ public String toString() {
     }
 
     public String toString(String sep) {
-        return from + sep + to + sep + getMaxWalkDistance() + sep + getDateTime() + sep
+        return from + sep + to + sep + getDateTime() + sep
                 + arriveBy + sep + optimize + sep + streetSubRequestModes.getAsStr() + sep
                 + getNumItineraries();
```

Comment:
Use the field here, not the accessor.

Reply:
This is not something that has been changed by this PR. If we want to access the field directly, we would need to use `new Date(dateTime * 1000)` instead of the method call in order to not change any functionality.